packages/discovery-styles/scss/components/search-facets/_search-facets.scss

```diff
@@ -8,6 +8,36 @@
   }
 }
 
+.#{$prefix}--search-facets {
+  .#{$prefix}--search-facet--category {
```

Comment:
not sure if this was copied, but if we are following BEM format, we should probably do something like `--search-facet__category` where `search-facet` is the block and `category` is the element. usually `--` indicates a modifier